resources/js/humhub.firebase.js

```diff
@@ -22,21 +33,78 @@ humhub.module('firebase', function (module, require, $) {
         }
     };
 
-    const afterServiceWorkerRegistration = function (registration) {
-        //console.log("After Service Worker Registration");
-        //console.log(registration);
+    /**
+     * Gets the content to display for notification permission status.
+     * @function
+     * @returns {string} The message corresponding to the current notification permission status.
+     */
+    const getNotificationPermissionContent = function () {
+        if (!("Notification" in window)) {
+            return module.text('status.not-supported');
+        }
+        switch (Notification.permission) {
+            case "granted":
+                return module.text('status.granted');
+            case "denied":
+                return module.text('status.denied');
+            default:
+                return module.text('status.default');
+        }
+    }
+
+    /**
+     * Shows the notification permission window and handles the user's response.
+     * @function
+     */
+    const showNotificationPermissionWindow = function () {
```

Comment:
@ArchBlood Is this method used?

Reply:
@luke- after closer inspection, you are correct this code is no longer needed, thank you for bring this to my attention!

Reply:
@ArchBlood Thanks!